brooklyn-docs/guide/ops/externalized-configuration.md

```diff
@@ -122,6 +122,19 @@ brooklyn.location.jclouds.aws-ec2.credential=$brooklyn:external("mysupplier", "a
 {% endhighlight %}
 
 
+## Referring to External Configuration in catalog items
+
+The same blueprint language DSL can be used within YAML catalog items. For example:
+
+    brooklyn.catalog:
+      id: com.example.myblueprint
+      version: 1.2.3
+      brooklyn.libraries:
+      - $brooklyn:external("mysupplier", "my-blueprint-lib")
+      item:
+        type: com.example.MyBlueprint
+
+
```

Comment:
Good thing to demonstrate,but example feels a bit contrived. Perhaps show a *location* catalog item, with user ID and credential externalised?